pkg/lib/cockpit-components-table.jsx

```diff
@@ -218,16 +218,16 @@ export const ListingTable = ({
                     {row.columns.map((cell, cellIndex) => {
                         const { key, ...cellProps } = cell.props || {};
                         const dataLabel = typeof cells[cellIndex] == 'object' ? cells[cellIndex].title : cells[cellIndex];
-
+                        const colKey = dataLabel || cellIndex;
```

Comment:
I suppose we don't need to be overly concerned about `""` labels? If we do, then this needs to be `??` instead of `||`.

Reply:
`??`?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing